_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -28,21 +28,41 @@ Defining a new detector involves naming the detector, selecting a data source an
 * Use the **Log type**, **Rule severity**, and **Source** dropdown menus to filter the rules you want to select from. 
 * Use the **Search** bar to search for specific rules.
 
-    To quickly select one or more known rules and dismiss others, first deselect all rules by moving the **rule name** toggle to the left, then search for your target rule names and select each individually by moving its toggle to the right.
+    To quickly select one or more known rules and dismiss others, first deselect all rules by moving the **Rule name** toggle to the left, then search for your target rule names and select each individually by moving its toggle to the right.
     {: .tip }
 
 1. In the **Detector schedule** section, set how often the detector will run. Specify a unit of time and a corresponding number to set the interval.
 1. Select the **Next** button in the lower-right corner of the screen to continue. The Configure field mapping page appears.
 
 ## Step 2. Make field mappings
```

Comment:
Maybe "Create field mappings"?

Reply:
oh and remember our style change to use gerunds for all section headings? So could be 
"Creating field mappings" then

Reply:
Agree with Fanit: we are standardizing gerunds are for the TOC/nav pane.